src/BayesFilters/include/BayesFilters/LTIStateModel.h

```diff
@@ -0,0 +1,44 @@
+#ifndef LTISTATEMODEL_H
+#define LTISTATEMODEL_H
+
+#include <Eigen/Dense>
+
+#include <BayesFilters/LinearStateModel.h>
+
+namespace bfl {
+    class LTIStateModel;
+}
+
+
+class bfl::LTIStateModel : public bfl::LinearStateModel
+{
+public:
+    LTIStateModel(Eigen::MatrixXf& transition_matrix,
```

Comment:
I generally prefer to have `const Eigen::Ref<const Eigen::MatrixXf>&` so that user can also pass blocks and other expressions. Is there a particular reason not ot do so?

Reply:
Ok! 👍